deps.ffmpeg/20-opus.zsh

```diff
@@ -57,6 +57,20 @@ config() {
         -DOPUS_FORTIFY_SOURCE=OFF
       )
       ;;
+    windows-arm64)
+      args+=(
+        -DOPUS_STACK_PROTECTOR=OFF
+        -DOPUS_FORTIFY_SOURCE=OFF
+        -DRUNTIME_CPU_CAPABILITY_DETECTION=OFF
+        -DOPUS_DISABLE_INTRINSICS=ON
+        # TODO: llvm-mingw ARM64 seh __try
```

Comment:
Remove unused code for PRs instead of using comments.

Reply:
ok